src/test/java/com/digirati/taxonomy/manager/lookup/persistence/SkosTranslatorTest.java

```diff
@@ -0,0 +1,203 @@
+                        .setAltLabel(
+                                "[{\"language\":\"fr\",\"value\":\"quatrieme\"},{\"language\":\"en\",\"value\":\"fourth\"}]");
+
+        ConceptSemanticRelationModel oneIsNarrowerThanTwo =
+                new ConceptSemanticRelationModel()
+                        .setSourceId("http://example.com/concept#1")
+                        .setTargetId("http://example.com/concept#2")
+                        .setRelation(SemanticRelationType.NARROWER)
+                        .setTransitive(false);
+        ConceptSemanticRelationModel twoIsRelatedToThree =
+                new ConceptSemanticRelationModel()
+                        .setSourceId("http://example.com/concept#2")
+                        .setTargetId("http://example.com/concept#3")
+                        .setRelation(SemanticRelationType.RELATED)
+                        .setTransitive(false);
+        ConceptSemanticRelationModel twoIsRelatedToFour =
+                new ConceptSemanticRelationModel()
+                        .setSourceId("http://example.com/concept#2")
+                        .setTargetId("http://example.com/concept#4")
+                        .setRelation(SemanticRelationType.RELATED)
+                        .setTransitive(false);
+        ConceptSemanticRelationModel threeIsBroaderThanOne =
+                new ConceptSemanticRelationModel()
+                        .setSourceId("http://example.com/concept#3")
+                        .setTargetId("http://example.com/concept#1")
+                        .setRelation(SemanticRelationType.BROADER)
+                        .setTransitive(false);
+        ConceptSemanticRelationModel threeIsNarrowerThanFour =
+                new ConceptSemanticRelationModel()
+                        .setSourceId("http://example.com/concept#3")
+                        .setTargetId("http://example.com/concept#4")
+                        .setRelation(SemanticRelationType.NARROWER)
+                        .setTransitive(false);
+        ConceptSemanticRelationModel fourIsBroaderThanThree =
+                new ConceptSemanticRelationModel()
+                        .setSourceId("http://example.com/concept#4")
+                        .setTargetId("http://example.com/concept#3")
+                        .setRelation(SemanticRelationType.BROADER)
+                        .setTransitive(false);
+
+        return new RdfModel(
+                Arrays.asList(one, three, two, four),
+                new ArrayList<>(),
+                Arrays.asList(
+                        twoIsRelatedToFour,
+                        twoIsRelatedToThree,
+                        oneIsNarrowerThanTwo,
+                        fourIsBroaderThanThree,
+                        threeIsNarrowerThanFour,
+                        threeIsBroaderThanOne));
+    }
+
+    private InputStream load(String rdfFileName) {
+        return FileManager.get().open("skos-translator-test-data/" + rdfFileName);
```

Comment:
`FileManager`? What's going on here?

Reply:
This is opening the file at `"skos-translator-test-data/" + rdfFileName` as an InputStream. Any specific objections to using `FileManager`?

Reply:
Yes, it has a whole load of code unrelated to opening files and the FileManager code is quite frankly garbage. `Files.newInputStream(...)` for something on the fileystem or ClassLoader.getResourceAsInputStream(...)` for something on the classpath is what should be used here.